agent/profile

```diff
@@ -24,11 +24,13 @@ function _pbench_setup() {
 	echo "                You will need to fix it before working with pbench." >&2
     else
 	local pdir
-	for pdir in ${prefix}/lib $(ls -1d ${prefix}/lib*/python3.*/site-packages); do
+	local NL=$'\n'
+	while read -r pdir; do
 	    if [[ ":${PYTHONPATH:-}:" != *":${pdir}:"* ]]; then
-		export PYTHONPATH=${pdir}${PYTHONPATH:+:${PYTHONPATH}}
+		PYTHONPATH=${pdir}${PYTHONPATH:+:${PYTHONPATH}}
 	    fi
-	done
+	done <<< "${prefix}/lib${NL}$(ls -1d ${prefix}/lib*/python3.*/site-packages)"
```

Comment:
I think this would be clearer if you created a file, putting the new lines where you want and reading from the file. As it stands, I think that every time I get to this point, I will stumble.

Reply:
I'd like to avoid using files.  In the profile setup especially.